swauth/authtypes.py

```diff
@@ -100,4 +110,56 @@ def match(self, key, creds):
         :param creds: User's stored credentials
         :returns: True if the supplied key is valid, False otherwise
         """
-        return self.encode(key) == creds
+
+        [type, rest] = creds.split(':')
```

Comment:
you are right, brackets are useless:

>>> a, b = "te:st".split(":")
>>> a
'te'
>>> b
'st'
>>> [a, b] = "te:st".split(":")
>>> a
'te'
>>> b
'st'
>>> (a, b) = "te:st".split(":")
>>> a
'te'
>>> b
'st'
>>> 

Should I create new PR for fixing this?